File: app/src/main/java/com/example/eugenia/game_2048/MainActivity.java
package com.example.eugenia.game_2048;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import java.lang.reflect.Field;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button startButton = findViewById(R.id.startButton);
        Button rulesButton = findViewById(R.id.rulesButton);
        Button exitButton = findViewById(R.id.exitButton);
        startButton.setOnClickListener(this);
        rulesButton.setOnClickListener(this);
        exitButton.setOnClickListener(this);

    }

    public void onClick(View view){
        switch (view.getId()) {
            case R.id.startButton: {
                Intent gameIntent = new Intent(MainActivity.this, GameActivity.class);
                startActivity(gameIntent);
                break;
            }
            case R.id.rulesButton: {
                Intent rulesIntent = new Intent(MainActivity.this, RulesActivity.class);
                startActivity(rulesIntent);
                break;
            }
            case R.id.exitButton: {
                finish();
                break;
            }
        }

    }


}

File: app/src/main/java/lestwald/insectkiller/MainActivity.kt
package lestwald.insectkiller

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.ImageButton

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val startButton = findViewById<ImageButton>(R.id.startButton)
        startButton.setOnClickListener {
            intent = Intent(this, GameFieldActivity::class.java)
            startActivity(intent) }
        val leaderBoardButton = findViewById<ImageButton>(R.id.leaderboardButton)
        leaderBoardButton.setOnClickListener {
            intent = Intent(this, LeaderBoardActivity::class.java)
            startActivity(intent) }
    }

}
